05-JWT-Basics/starter/middleware/error-handler.js

```diff
@@ -1,9 +1,12 @@
-const CustomAPIError = require('../errors/custom-error')
+const CustomAPIError = require('../errors/custom-error');
+const {StatusCodes} = require('http-status-codes');
+
+
```

Comment:
You're also going to want to `console.log(err)` in here so you can see what is going wrong in the console (or log files if this is deployed to a server somewhere).